examples/submit_custom_vocabularies.js

```diff
@@ -0,0 +1,48 @@
+const revai = require('revai-node-sdk');
+const token = require('./config/config.json').access_token;
+
+(async () => {
+    // Initialize your client with your revai access token
+    var client = new revai.RevAiCustomVocabulariesClient(token);
+
+    cv_submission = await client.submitCustomVocabularies([{
+            phrases: [
+                "enter",
+                "your",
+                "vocabularies",
+                "here"
+            ]
+        }])
+
+    console.log(`Job Id: ${cv_submission.id}`);
+    console.log(`Status: ${cv_submission.status}`);
+    console.log(`Created On: ${cv_submission.created_on}`);
+    /**
+     * Waits 5 seconds between each status check to see if the custom vocabulary
+     * is complete. note: polling for custom vocabulary status is not recommended in a
+     * non-testing environment. Use the callback_url option
+     * (see: https://www.rev.ai/docs#section/Node-SDK) to receive the response
+     * asynchronously on custom vocabulary completion.
+     */
+    while(
+        (cv_submission = await client.getCustomVocabularyInformation(cv_submission.id)).status
+            == revai.CustomVocabularyStatus.InProgress
+    )
+    {
+        console.log(`Job ${cv_submission.id} is ${cv_submission.status}`);
```

Comment:
I think we said we wanted to stop calling it a job.
```suggestion
        console.log(`Custom Vocab ${cv_submission.id} is ${cv_submission.status}`);
```